buildSrc/src/main/java/saros/gradle/intellij/SarosIntellijPlugin.java

```diff
@@ -0,0 +1,46 @@
+package saros.gradle.intellij;
+
+import org.gradle.api.Plugin;
+import org.gradle.api.Project;
+import org.gradle.api.Task;
+import org.gradle.api.tasks.TaskContainer;
+import org.jetbrains.intellij.IntelliJPlugin;
+
+/**
+ * Plugin that configures the gradle intellij plugin. The used plugin version is specified by the
+ * build.gradle file in the buildSrc dir.
+ *
+ * <p>The main purpose of this plugin is to set default configs and to add additional tasks to the
+ * runIde task (provided by the intellij plugin) in order to allow us to run multiple intellij
+ * instances simultaneously with runIde. Without our custom tasks the second runIde call would
+ * fails, because the config directory (within the build dir of the corresponding project) is
+ * already in use. Therefore we enhance the config directory selection. See {@code
```

Comment:
I think you meant to use a reference here instead of just displaying it as 'code'.

```suggestion
 * already in use. Therefore we enhance the config directory selection. See {@link
```